src/components/card/DiagramsCard.vue

```diff
@@ -11,37 +11,49 @@
         <q-btn
           outline
           no-caps
-          class="q-ml-xl"
+          label="Create diagram"
           color="primary"
           icon="fa-solid fa-layer-group"
-          :label="$t('actions.models.add.button.name')"
-          :title="$acl.role('create-diagram-from-template')
-            ? $t('actions.models.add.button.title')
-            : $t('errors.permissionsDenied')"
-          data-cy="add-model-button"
-          :disable="!$acl.role('create-diagram-from-template')"
-          @click="DrawerEvent.next({
-            key: 'CreateDiagram',
-            type: 'open',
-          })"
-        />
-        <q-btn
-          outline
-          no-caps
-          class="q-ml-xl"
-          color="primary"
-          icon="fa-solid fa-plus"
-          :label="$t('actions.models.create.button.name')"
-          :title="$acl.role('create-diagram')
-            ? $t('actions.models.create.button.title')
-            : $t('errors.permissionsDenied')"
-          data-cy="create-model-button"
-          :disable="!$acl.role('create-diagram')"
-          @click="DialogEvent.next({
-            type: 'open',
-            key: 'CreateModel',
-          })"
-        />
+          data-cy="create-diagram-list-button"
+        >
+          <q-menu auto-close>
+            <q-list
+              style="min-width: 100px"
+            >
+              <q-item
+                clickable
+                :label="$t('actions.models.create.button.scratch.name')"
+                :title="$acl.role('create-diagram')
```

Comment:
better put the ternary out of the template part, no ?

Reply:
We did it this way in every other pages.

Reply:
Agree with @aboubakar14